From: Debian sysvinit maintainers
 <debian-init-diversity@chiark.greenend.org.uk>
Date: Fri, 5 Apr 2019 20:32:40 +0000
Subject: _m68k_alignment

Purpose: Get insserv building on m68k, dropping some problematic
         alignments (error: requested alignment is not a power of 2)
Fixes:   #493637
Status:  Not yet submitted upstream.

Status:  Not yet submitted upstream.
---
 insserv.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/insserv.c b/insserv.c
index f48bc68..849b890 100644
--- a/insserv.c
+++ b/insserv.c
@@ -71,6 +71,10 @@
 #include "listing.h"
 #include "systemd.h"
 
+#ifdef __m68k__ /* Fix #493637 */
+#  define aligned(a)
+#endif
+
 #if defined _XOPEN_SOURCE && (_XOPEN_SOURCE - 0) >= 600
 # ifndef POSIX_FADV_SEQUENTIAL
 #  define posix_fadvise(fd, off, len, adv)  (-1)
